### ISSUE TITLE
How to normalize a tensor to [-1,1] in tensorflow??? 

### ISSUE THREAD

**Issue by longmalongma** [yenchenlin/nerf-pytorch#12]
How to normalize a tensor to [-1,1] in tensorflow???

**Comment by yenchenlin** [comment]
Please ask the question on StackOverflow.
